packages/building-utils/index-html/create-index-html.js

```diff
@@ -51,6 +51,7 @@ const { createContentHash, cleanImportPath, polyfillFilename } = require('./util
  * @property {boolean} [coreJs] whether to polyfill core-js polyfills
  * @property {boolean | string} [regeneratorRuntime] whether to add regenerator runtime
  * @property {boolean} [webcomponents] whether to polyfill webcomponents
+ * @property {boolean} [shadyCSS] whether to polyfill shadyCSS
```

Comment:
Shady CSS is loaded as part of `webcomponentjs`, right? This is only adding the custom style interface? So maybe we should call it `shadyCssCustomStyle`?

Reply:
Good point. It gets confusing fast with the separation of all the poly fills, but you're right, custom-style-interface is what we want FROM shadyCSS (I don't understand why it's a separate thing not loaded by webcomponentsjs?). https://github.com/open-wc/open-wc/pull/1292/files/d36a90f92959dae3474ef3b6cb92610bbe595159#diff-0a09ec55ca2108cb9656fd314415a8e8R187

Reply:
I *think* shady css is loaded for polyfilling styles in shadow dom, but only the parts that are strictly necessary by default.